coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -4882,3 +4882,24 @@ def test_sum(self, backend, input_dtype):
             input_as_shape=False,
             backend=backend,
         )
+
+
+class TestHannWindow(TorchBaseTest):
+    @pytest.mark.parametrize(
+        "backend, window_length",
+        itertools.product(
+            backends,
+            [1, 3, 6],
+        ),
+    )
+    def test_hann_window(self, backend, window_length):
+        class HannWindowModel(nn.Module):
+            def forward(self, x):
+                return torch.hann_window(window_length)
+
+        input_shape = np.random.randint(low=2, high=6, size=(window_length,))
```

Comment:
The range of 2 to 6 does not contain many int values. I think we should increase this range quite a bit.